views/account/settings/main.js

```diff
@@ -146,6 +146,8 @@ const displayChanges = (fontSize, fontFamily, theme, contrast) => {
   const div = document.getElementById("body"); // Get element from DOM
```

Comment:
I know you're still working on this, but I want to let you know early about this code. From the theme flashing bug fix, none of this code is in this file anymore so you'll probably have conflicts.

Reply:
Yup! I'm currently refactoring it since I saw that you had moved it out of the file.

Shouldn't be too bad though.

Thank you for the heads up!

Reply:
No problem chief, figured I'd try and prevent a headache later!